app/gateway/2.7.x/developer-portal/theme-customization/migrating-templates.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Migrating Templates Between Workspaces
+badge: enterprise
+---
+
+The template styling from your default Dev Portal doesn't automatically apply to new Dev Portals you create in other workspaces. However, with the [Portal CLI](/gateway/latest/developer-portal/helpers/cli/), you can move over templates in a few steps.
+
+## Prerequisites
+
+* Install [Portal CLI](/gateway/latest/developer-portal/helpers/cli/) and set up the [configuration](/gateway/latest/developer-portal/helpers/cli/#usage).
+
+## Apply customization from default Dev Portal
+
+To apply customizations from one Dev Portal to another, do the following:
+
+1. From your Kong Manager, create a new Workspace and enable a new Dev Portal. In the rest of this document, the new Workspace name will be referred to as `{WORKSPACE_NAME}`.
+
+2. Git clone `https://github.com/Kong/kong-portal-templates`.
```

Comment:
(I can't leave this feedback as suggestions, since it includes backticks)

Please add descriptions for what each step is doing and move the command itself into a codeblock. For this one:

> Clone the Portal templates repository:
> ```sh
> git clone https://github.com/Kong/kong-portal-templates
> ```

This might feel like overkill, but you'd be surprised how often people run into problems with stuff like this.